kombu/utils/json.py

```diff
@@ -12,19 +12,12 @@
     class DjangoPromise:
         """Dummy object."""
 
-try:
-    import json
-    _json_extra_kwargs = {}
 
-    class _DecodeError(Exception):
-        pass
-except ImportError:                 # pragma: no cover
-    import simplejson as json
-    from simplejson.decoder import JSONDecodeError as _DecodeError
-    _json_extra_kwargs = {
-        'use_decimal': False,
-        'namedtuple_as_object': False,
-    }
+_json_extra_kwargs = {}
```

Comment:
This variable seems unnecessary. Since it is private -- maybe you can remove it as well ?

Reply:
Removed and usage replaced with or